java/java-guestbook/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.1/apache-maven-3.6.1-bin.zip
```

Comment:
Will we need to update this path periodically when maven gets updated?

Reply:
Correct, if we want Maven version update for some reason. Maven is very stable though.